src/pandas_profiling/config.py

```diff
@@ -305,11 +306,22 @@ class Config:
         "heatmap": True,
     }
 
+    correlation_table: bool = True
+
     correlations: Dict[str, Correlation] = {
         "auto": Correlation(key="auto"),
+        "spearman": Correlation(key="spearman"),
+        "pearson": Correlation(key="pearson"),
+        "phi_k": Correlation(key="phi_k"),
+        "cramers": Correlation(key="cramers"),
+        "kendall": Correlation(key="kendall"),
     }
 
-    correlation_table: bool = True
+    correlations["auto"].calculate = True
+    correlations["pearson"].calculate = False
+    correlations["phi_k"].calculate = False
+    correlations["cramers"].calculate = False
+    correlations["kendall"].calculate = False
```

Comment:
you can just do `"spearman": Correlation(key="spearman", calculate=True)`

Reply:
I was following what was done previously and kept the same logic so I don't need to change both (check the history for the PR).